eo-runtime/src/main/java/org/eolang/PhPackage.java

```diff
@@ -49,7 +49,7 @@ final class PhPackage implements Phi {
     @Override
     public Attr attr(final String name) {
         final String abs = String.format("%s.%s", this.pkg, name);
-        final String target = abs.replaceAll("(^|\\.)([^.]+)", "$1EO$2");
+        final String target = abs.replaceAll("(^|\\.)([^.]+)", "$1EO$2").replace("-", "_");
```

Comment:
@yegor256 When the `name` parameter of this method is `org.eolang.as-phi`, after replacing it would be `EOorg.EOeolang.EOas-phi`. But there is no such class in `EOorg.EOeolang` package. We have `EOorg.EOeolang.EOas_phi`, so I decided to replace `-` to `_`.